src/jit/compiler.h

```diff
@@ -6361,6 +6361,23 @@ public :
 
     GenTreePtr                  eeGetPInvokeCookie(CORINFO_SIG_INFO *szMetaSig);
 
+    inline size_t               eeGetPageSize() const
+    {
+        return CORINFO_PAGE_SIZE;
+    }
+
+    inline size_t               getVeryLargeFrameSize() const
+    {
```

Comment:
Not only should there be a function comment here, but you need to adjust the comment from the location where you took this code (line 9121)